beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/service/PathSchedulerApiary.java

```diff
@@ -18,39 +18,57 @@
 import static java.lang.String.format;
 
 import java.io.IOException;
+import java.util.List;
 import java.util.Optional;
 
+import static com.expediagroup.beekeeper.core.model.LifeCycleEventType.EXPIRED;
+import com.expediagroup.beekeeper.core.repository.HousekeepingPathRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 import com.expediagroup.beekeeper.core.error.BeekeeperException;
 import com.expediagroup.beekeeper.core.model.HousekeepingPath;
 import com.expediagroup.beekeeper.scheduler.apiary.messaging.PathEventReader;
-import com.expediagroup.beekeeper.scheduler.apiary.model.PathEvent;
+import com.expediagroup.beekeeper.scheduler.apiary.model.PathEvents;
 import com.expediagroup.beekeeper.scheduler.service.SchedulerService;
 
 @Component
 public class PathSchedulerApiary {
```

Comment:
That makes sense, having a better separation or responsibility for the orchestrator should make that easier to follow. Currently going through and trying to decouple that based on these comments, as well as adding the update rather than del/create